roles/bionic/postgresql10/tasks/install.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Adding APT repository key
+  apt_key: 
+    id: "{{ postgresql_apt_key_id }}" 
+    url: "https://www.postgresql.org/media/keys/{{ postgresql_apt_key_id}}.asc"
+
+- name: Add PostgreSQL official APT repository
+  apt_repository: 
+    repo: "deb http://apt.postgresql.org/pub/repos/apt/ {{ansible_distribution_release}}-pgdg main"
+
+- name: Install apt packages
+  apt: 
+    pkg: "{{ item }}"
+    state: present
+  with_items:
+    - "postgresql-{{ postgresql_version }}"
+    - python-psycopg2
+    - libpq-dev
+
+- name: Add postgis repo
+  apt_repository:
+    repo: ppa:ubuntugis/ubuntugis-unstable
+    state: present
+
+- name: Install postgresql-{{ postgresql_version }}-postgis-{{ postgis_version }}
+  apt:
+    pkg: postgresql-{{ postgresql_version }}-postgis-{{ postgis_version }}
+    state: present
+
```

Comment:
this is not the correct format.  Look at the other examples and no need to start postgres here because we're not creating the extension if not exists because it's dropped if the DB is dropped.

Reply:
OK, if not require to use create extension statement, it can be removed.